js/lib/appUrlUtil.js

```diff
@@ -124,25 +124,25 @@ module.exports.isTargetAboutUrl = function (input) {
 }
 
 /**
- * Gets a local URL that renders the given magnet:// URL
```

Comment:
Most of the changes here are to match the style of the existing about url functions